docs/contributtion/experimental-code.md

```diff
@@ -0,0 +1,167 @@
+# Experimental Code
+
+The team occasionally will author code, or accept contributions, that is
+considered experimental or unstable. The goal for this code is to ship it as
+unstable for sponsor product to leverage. During this time, the team can get
+feedback around what is working and what does not work so that changes can be
+made before an official release.
+
+Experimental code is made available in the following ways:
+
+- Components and exports prefixed with `UNSTABLE_`
+- Feature flags
+- Expanding existing APIs
+
+In all cases, experimental code should be treated as unstable and can change
+between release versions for the package that it is being imported from.
+
+- The API is not fixed, and is likely to change
+- The API is not bound by semver
+- The component, export, feature, etc. may change, be renamed, or removed in the
+  future without warning
+
+## Components and exports prefixed with `UNSTABLE_`
+
+For experimental or unstable exports, we use the `UNSTABLE_` prefix. For
+example:
+
+```js
+// An unstable method
+function UNSTABLE_layout() {
+  // ...
+}
+
+// An unstable variable
+const UNSTABLE_meta = {
+  // ...
+};
+
+// An unstable component will retain its name, specifically for things like
+// the rules of hooks plugin which depend on the correct casing of the name
+function ExampleComponent(props) {
+  // ...
+}
+
+// However, when we export the component we will export it with the `UNSTABLE_`
+// prefix. (Similar to React.unstable_Suspense, React.unstable_Profiler)
+// however we will use it in all caps format
+export { default as UNSTABLE_ExampleComponent } from './components/ExampleComponent';
```

Comment:
oh wait, that means I have it wrong? Does it mean that everything in the Divider component #1456 should be just called Divider (files, folders, etc) and only in the index.ts of the component should I use this named export?

Reply:
I will rewrite this copy.